easybuild/tools/toolchain/mpi.py

```diff
@@ -191,7 +191,8 @@ def mpi_cmd_for(self, cmd, nr_ranks):
             tmpdir = tempfile.mkdtemp(prefix='eb-mpi_cmd_for-')
 
             # set temporary dir for mdp
-            env.setvar('I_MPI_MPD_TMPDIR', tmpdir)
+            # note: this needs to be kept *short*, to avoid mpirun failing with "socket.error: AF_UNIX path too long"
+            env.setvar('I_MPI_MPD_TMPDIR', tempfile.gettempdir())
```

Comment:
no, and it seems like the limit is system-dependent:

```
The mpd2.console_* file full path length can be limited in some operating systems. You hit
this limitation if you get the following diagnostic message: socket.error: AF_UNIX path
too long. Decrease the length of the <directory name> string to avoid this issue. 
```